chipsec/modules/common/uefi/access_uefispec.py

```diff
@@ -64,6 +64,7 @@ def __init__(self):
 
         self.uefispec_vars = {
             # From UEFI Spec Table 11 "Global Variables"
+            # Updated to version 2.10 Table 3.3 "Global Variables"
```

Comment:
```suggestion
            # Updated to version 2.10 Section 3.3 Table 3.1 "Global Variables"
```
Might also be worth putting a link in there? https://uefi.org/specs/UEFI/2.10/03_Boot_Manager.html#globally-defined-variables